### ISSUE TITLE
Migrations for enum attributes are wrong

### ISSUE THREAD

**Issue by dj-hedgehog** [strapi/strapi#109]
I've got this attribute in a model:

```
"state": {
    "type": "enum",
    "values": [
        "queued",
        "pending",
        "finished",
    ],
    "defaultTo": "queued"
}
```

Upon creating the migration this is created from its template:

```
table.enu('state', queued, pending, finished)
        .defaultTo('queued');
```

but it should be this:

```
table.enu('state', ['queued', 'pending', 'finished'])
        .defaultTo('queued');
```

**Comment by Aurelsicoko** [comment]
@dj-hedgehog: Could you submit a PR for that?

**Comment by dj-hedgehog** [comment]
The migration-generation is completely broken (for me) currently. Every since @loicsaintroch generated working examples for the models I started with I wrote them by hand.

The question is how we want to deal with this? I'd like to try to fix migrations within the next weeks but bookshelf etc. needs to be put into its own plugin anyways. There are two problems:

a) I fix the `strapi-generate-migrations` BEFORE you put it into a new v3 plugin:
you have to wait for me to fix it first before you can do it

b) I fix the `strapi-generate-migrations` AFTER you put it into a new v3 plugin:
I'll probably need access to v3 so I can fix it.

Decide for yourself what you'd prefer.

**Comment by Aurelsicoko** [comment]
`strapi-bookshelf` will not be a plugin. It already a hook available as `strapi-mongoose` in our new mono-repository structure (already available for the v2.0). As I said, I will publish the v3.0 asap. Before that, we have to migrate the repositories to another organisation because the `wistityhq` will no longer exist. You will be able to access to the `strapi-generation-migrations` and make PR on it. It has nothing to do with the plugins.

Do migrations are broken on the enum attribute only or on others things too?
